engine/src/test/scala/cromwell/util/SampleWdl.scala

```diff
@@ -2317,7 +2317,7 @@ task shouldCompleteFast {
         |    Int a
         |    command {
         |        echo "The number was: ${a}"
-        |        sleep 0.2
+        |        sleep 0.15
```

Comment:
Turns out Fail Fast-ish lets there be a race condition here, if the failing and successful tasks take the same time, sometimes the successful task's successor has already started by the time the failure is noticed. So this increases (from 0) the gap between the preceding failure and success.